test/DocumentFormat.OpenXml.Tests/OpenXmlDomTest/OpenXmlSimpleValueTest.cs

```diff
@@ -84,7 +85,13 @@ public void BooleanValueTest()
             Log.Comment("Verifying reference type behavior...");
             objA = new BooleanValue(validValue);
             objB = new BooleanValue(validValue);
-            Log.VerifyFalse(object.Equals(objA, objB), "Two instances with same value but constructed separated are Equal.");
+
+            // This is the previous assertion, which fails due to the proposed change:
```

Comment:
Can you remove these comments and update it to use `Assert.Equal` instead - the `Log` file system is something I'd like to move away from as XUnit provides most of the testing stuff (and helpful diagnostics) we need

Reply:
Sounds good. I'm not saying remove all the Log.Verify[...] calls, just switch to XUnit for any new code :) I've been chipping away at the removal of the old Log infrastructure for a while now... I definitely don't expect that in this PR :)